src/__tests__/orders.test.js

```diff
@@ -0,0 +1,194 @@
+/**
+ * @file This file contains tests for submitMutliOrder
+ *
+ */
+
+import { createMockedDataLoaders } from "../datasourceLoader";
+import { performTestQuery } from "../utils/utils";
+
+const userParameters = {
+  userName: "Test Testesen",
+  userMail: "test@dbc.dk",
+};
+
+const query = `
+          mutation Example_SubmitMultipleOrders($input: SubmitMultipleOrdersInput!) {
+            submitMultipleOrders(input: $input, dryRun: false) {
+              failedAtCreation
+              successfullyCreated
+              status
+              ok
+            }
+          }`;
+
+const context = {
+  datasources: createMockedDataLoaders(),
+  accessToken: "DUMMY_TOKEN",
+  user: {
+    userId: "some-id",
+    loggedInAgencyId: "715100",
+    municipalityAgencyId: "100200",
+  }, //hvis jeg ikke bruger userinfo, så kan jeg sætter user her
+  smaug: {
+    app: { id: "app-name", ips: ["1.1.1.1"] },
+    orderSystem: "bibliotekdk_21",
+  },
```

Comment:
Hvor meget af smaug er det som er udfaset?

Reply:
Den har jeg bare copypastet fra order tests'ne, som morten har tilføjet for 3 måneder siden. Jeg antager de burde lave det samme. Jeg kender ikke til smaug-udfasning. 
Order koden kaster i hvert fald en fejl "Unexpected error value: "invalid smaug configuration [orderSystem]"], måske er der noget der mangler " når jeg ikke har den med